.github/workflows/ci.yml

```diff
@@ -2,20 +2,21 @@ name: CI
 on:
     push:
         branches:
-            - main
+            - '*'
     pull_request:
 jobs:
     check:
         name: Check
         runs-on: ubuntu-latest
         timeout-minutes: 10
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v2.3.4
```

Comment:
I don't think we should pin to specific patch versions (same for the other non-major updates in this file). Instead we should keep `checkout@v2` so it automatically uses the latest non-breaking version.

Reply:
Reverted.